### ISSUE TITLE
Trajectory data without data variable fail to read as point data 

### ISSUE THREAD

**Issue by rsignell-usgs** [Unidata/thredds#446]
Many ocean drifters just report report longitude and latitude at periodic times along their trajectory - there are no temperature, salinity or other sensors on them.

It is expected that trajectory DSG data should fail if there isn't an additional data variable present?

If I open this OPeNDAP data URL as point data in ToolsUI, I get an error:
http://thredds.socib.es/thredds/dodsC/drifter/surface_drifter/drifter_svpf001-ime_svpf001/L0/2012/dep0001_drifter-svpf001_ime-svpf001_L0_2012-10-25.nc

However, if I add a simple dummy temperature variable:
```
  <ncml:variable name="T" shape="time" type="double">
    <ncml:attribute name="standard_name" value="sea_water_temperature" />
    <ncml:attribute name="units" value="degrees_Celcius" />
    <ncml:attribute name="values" start="10.0" increment="0.001"/>
    <ncml:attribute name="_FillValue" type="double" value="-99999.9" />
    <ncml:attribute name="coordinates" value="time DEPTH LAT LON" />
  </ncml:variable>
```

then the NcML dataset opens just fine in ToolsUI:
```
<?xml version="1.0" encoding="UTF-8"?>
<ncml:netcdf xmlns:ncml="http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2" location="dods://thredds.socib.es/thredds/dodsC/drifter/surface_drifter/drifter_svpf001-ime_svpf001/L0/2012/dep0001_drifter-svpf001_ime-svpf001_L0_2012-10-25.nc">
 
  <ncml:variable name="T" shape="time" type="double">
    <ncml:attribute name="standard_name" value="sea_water_temperature" />
    <ncml:attribute name="units" value="degrees_Celcius" />
    <ncml:attribute name="values" start="10.0" increment="0.001"/> 
    <ncml:attribute name="_FillValue" type="double" value="-99999.9" />
    <ncml:attribute name="coordinates" value="time DEPTH LAT LON" />
  </ncml:variable>
</ncml:netcdf>
```
Is this expected behavior?

**Comment by lesserwhirls** [comment]
Hi @rsignell-usgs,

We've been discussing this at our weekly THREDDS meeting this morning, and we were wondering if you could post this to the CF list. I am wondering if the case where only coordinate variables exist in the file is addressed by the DSG types.

Thanks!

**Comment by rsignell-usgs** [comment]
Okay

On Thu, Feb 25, 2016 at 2:05 PM, Sean Arms <notifications@github.com> wrote:

> Hi @rsignell-usgs <https://github.com/rsignell-usgs>,
>
> We've been discussing this at our weekly THREDDS meeting this morning, and
> we were wondering if you could post this to the CF list. I am wondering if
> the case where only coordinate variables exist in the file is addressed by
> the DSG types.
>
> Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Unidata/thredds/issues/446#issuecomment-188933356>.
>



-- 
Dr. Richard P. Signell   (508) 457-2229
USGS, 384 Woods Hole Rd.
Woods Hole, MA 02543-1598

**Comment by JohnLCaron** [comment]
can we get sample file?

**Comment by lesserwhirls** [comment]
Here is the catalog:

http://thredds.socib.es/thredds/catalog/drifter/surface_drifter/drifter_svpf001-ime_svpf001/L0/2012/catalog.html?dataset=drifter/surface_drifter/drifter_svpf001-ime_svpf001/L0/2012/dep0001_drifter-svpf001_ime-svpf001_L0_2012-10-25.nc

and here is the httpServer link:

http://thredds.socib.es/thredds/fileServer/drifter/surface_drifter/drifter_svpf001-ime_svpf001/L0/2012/dep0001_drifter-svpf001_ime-svpf001_L0_2012-10-25.nc

(~ 8kb)

Just noticed that the DEPTH variable is not a function of time. Is that correct? It's just a scalar variable?

**Comment by rsignell-usgs** [comment]
Yes, this is a drifter that floats at the surface of the ocean.   So the depth is constant.
